ui/app/src/state/actions/preview.ts

```diff
@@ -64,6 +64,7 @@ export const CONTENT_TYPE_RECEPTACLES_REQUEST = 'CONTENT_TYPE_RECEPTACLES_REQUES
 export const CONTENT_TYPE_RECEPTACLES_RESPONSE = 'CONTENT_TYPE_RECEPTACLES_RESPONSE';
 export const SCROLL_TO_RECEPTACLE = 'SCROLL_TO_RECEPTACLE';
 export const CLEAR_HIGHLIGHTED_RECEPTACLES = 'CLEAR_HIGHLIGHTED_RECEPTACLES';
+export const SCROLL_TO_ELEMENT = 'SCROLL_TO_ELEMENT';
```

Comment:
Since we're also animating, etc use a more specific and semantic action type: CONTENT_TREE_FIELD_SELECTED